opta/nice_subprocess.py

```diff
@@ -22,79 +24,133 @@ def nice_run(  # type: ignore # nosec
     capture_output: bool = False,
     timeout: Optional[float] = None,
     exit_timeout: Optional[float] = None,
+    tee: Optional[bool] = True,
     check: bool = False,
     **kwargs,
 ) -> CompletedProcess:
-    """Run command with arguments and return a CompletedProcess instance.
-
-    The returned instance will have attributes args, returncode, stdout and
-    stderr. By default, stdout and stderr are not captured, and those attributes
-    will be None. Pass stdout=PIPE and/or stderr=PIPE in order to capture them.
-
-    If check is True and the exit code was non-zero, it raises a
-    CalledProcessError. The CalledProcessError object will have the return code
-    in the returncode attribute, and output & stderr attributes if those streams
-    were captured.
-
-    If timeout is given, and the process takes too long, a TimeoutExpired
-    exception will be raised.
-
-    There is an optional argument "input", allowing you to
-    pass bytes or a string to the subprocess's stdin.  If you use this argument
-    you may not also use the Popen constructor's "stdin" argument, as
-    it will be used internally.
-
-    By default, all communication is in bytes, and therefore any "input" should
-    be bytes, and the stdout and stderr will be bytes. If in text mode, any
-    "input" should be a string, and stdout and stderr will be strings decoded
-    according to locale encoding, or by "encoding" if set. Text mode is
-    triggered by setting any of text, encoding, errors or universal_newlines.
-
-    The other arguments are the same as for the Popen constructor.
-    """
```

Comment:
If this docstring is no longer accurate, it should be updated instead of removed.